odoo/addons/test_new_api/models/test_new_api.py

```diff
@@ -2290,3 +2290,25 @@ class Test_New_ViewStrId(models.Model):
     _table_query = "SELECT 'hello' AS id, 'test' AS name"
 
     name = fields.Char()
+
+
+class Test_New_ApiCreatePerformance(models.Model):
+    _name = _description = 'test_new_api.create.performance'
+
+    one_bool = fields.Boolean()
+    name = fields.Char()
+    name_store = fields.Char(compute='_compute_name_store', store=True)
+    line_o2m_ids = fields.Many2many('test_new_api.create.performance.line', 'perf_id')
+    line_m2m_ids = fields.Many2many('test_new_api.create.performance.line', 'smaller_table_plz')
```

Comment:
May I suggest better names?
* `some_bool` is more English than `one_bool`, but I even suggest something like `confirmed` to make the test code more intuitive;
* `name_store` suggests it is the same as `name` but stored, but `name` is stored already :upside_down_face: Why not an integer field instead, like `name_changes` which is incremented in its compute method?
* for one2many fields, I suggest `line_ids` (usually not shared across records)
* for many2many fields, I suggest `tag_ids` (usually shared across records)